vic/drivers/cesm/bld/build_vic_namelist

```diff
@@ -91,7 +91,8 @@ def copy_to_rundir(grid_config, caseid, rundir, casedocs):
 
         # copy file
         copy_clean_vic_config(grid_config[filekey], dst_file,
-                              header=header, rundir=rundir, **grid_config)
+                              header=header, rundir=rundir, 
+			       caseid=caseid, **grid_config)
```

Comment:
can you fix the indentation of this line?

Reply:
Done.